test/spec.index.js

```diff
@@ -359,6 +384,117 @@ describe('Template Mixins', function () {
                 }));
             });
 
+            it('autocomplete should have a sufix of -day -month and -year', function () {
```

Comment:
You could wrap these new tests in  a `describe` block so that `it`... make sense.